### PR TITLE
 minor edits

### PR DESCRIPTION
**Pull request by rosemarymarano**

### CODE REVIEW

File: todo-list/README.md [hunk 1]
```diff
@@ -24,14 +24,13 @@ Create a sample domain that represents your on-premises WebLogic Server domain:
 2. [Create a WebLogic Server domain](#create-a-new-weblogic-server-domain).
     - Create the sample domain that represents the on-premises domain.
     - Configure a data source to access the MySQL database.
-    - Domain Limitations:
-        - There are 2 supported domain types, single server and single cluster
-        - Domains must have the default value of AdminServer
+    - Be aware of these domain limitations:
+        - There are two supported domain types, single server and single cluster.
+        - Domains must have the default value, `AdminServer`.
```
Comment (pfmackin) [inline todo-list/README.md:29]: What field does this default value of 'AdminServer' refer to?   I don't know what this means.

Comment (rosemarymarano) [reply todo-list/README.md:29]: We will have to check with @ddrechse or @ddsharpe to be sure, but my guess is that it is the default name value for the Admin Server.

Comment (ddsharpe) [inline todo-list/README.md:29]: I believe this should read:
Domains must use `AdminServer` as the value for `AdminServerName`.  (This is the default value)

Comment (rosemarymarano) [reply todo-list/README.md]: Fixed, thank you for pointing this out, @pfmackin

Comment (ddsharpe) [reply todo-list/README.md]: This sentence does not make sense now.

Comment (rosemarymarano) [reply todo-list/README.md]: @ddsharpe I believe that you are not looking at the final wording. It is , "Domains must use the default value, `AdminServer`, for `AdminServerName`." I slightly re-worded your suggestion, "Domains must use AdminServer as the value for AdminServerName. (This is the default value)" If you feel strongly about it, I will change it to your suggested text (I am not opposed to it.)

**Review by ddsharpe — APPROVED** [review]

**Pull request merged**